imports/plugins/included/product-admin/client/hocs/withVariantForm.js

```diff
@@ -175,19 +185,35 @@ const wrapComponent = (Comp) => (
         return;
       }
 
-      Meteor.call("products/updateProductField", variantId, fieldName, value, (error) => {
-        if (error) {
-          Alerts.toast(error.message, "error");
-        }
-
-        if (fieldName === "inventoryPolicy") {
-          this.updateInventoryPolicyIfChildVariants(variant);
-        }
+      // Check to see if field has been updated.
+      // `onProductFieldSave` runs onChange AND onBlur,
+      // so there isn't always an update to a value when run with `onBlur`,
+      // and this mutation might not need to run
+      if (variant[fieldName] !== value) {
+        const { client } = this.props;
+        const [opaqueProductId, opaqueShopId] = await getOpaqueIds([
+          { namespace: "Product", id: variant._id },
+          { namespace: "Shop", id: Reaction.getShopId() }
+        ]);
+
+        try {
+          await client.mutate({
+            mutation: updateProductField,
+            variables: {
+              input: {
+                field: fieldName,
+                shopId: opaqueShopId,
+                productId: opaqueProductId,
+                value
+              }
+            }
+          });
 
-        if (fieldName === "lowInventoryWarningThreshold") {
-          this.updateLowInventoryThresholdIfChildVariants(variant);
```

Comment:
@aldeed I believe this, and also the other check for `updateInventoryPolicyIfChildVariants`, is now being done in the simple-inventory plugin, with the productConfiguration, can you confirm that's the case? https://github.com/reactioncommerce/reaction/blob/refactor-kieckhafer-convertCatalogMethodsToGQL-products/updateProductField/imports/node-app/plugins/simple-inventory/mutations/updateSimpleInventory.js#L54